### DISCUSSION TITLE
Chrome: Zum Startbildschirm hinzufügen

### DISCUSSION THREAD

**Discussion by quargh** [neuefische/questions#128] [Web]
Hello, 

wenn ich im mobilen Chrome-Browser (in meinem Fall Android) unter Menu -> zum Startbildschirm hinzufügen klicke, wie folgt:

![Screenshot_2022-08-30-20-34-21-274_com android](https://user-images.githubusercontent.com/30600277/187518205-658017a0-d71f-45fd-870d-d08d50888c78.png)

... erscheint auf dem Homescreen im Fall einer create-React Anwendung ein Icon mit React Logo. Im Fall unserer Capstone Next Anwendung ein Standard Icon mit V wie Vercel:

![icons](https://user-images.githubusercontent.com/30600277/187519437-19981451-f00a-499b-883a-1b82252f678e.png)

Außerdem wird die React App on Klick auf das Icon im Fullscreen-Mode angezeigt, mit ausgeblendeter URL-Eingabeleiste und die Android Navigationsleiste unten wird schwarz:

![Screenshot_2022-08-30-20-33-11-133_com android chrome](https://user-images.githubusercontent.com/30600277/187519924-60b3e953-e958-4780-a453-ccfad59b8687.jpg)

Im Fall der Next-Anwendung ist es eigentlich nur eine Art Lesezeichen Verknüpfung, die URL-Zeile ist weiterhin zu sehen und die Navigationsleiste unten ist weiß:

![Screenshot_2022-08-30-20-34-27-405_com android chrome](https://user-images.githubusercontent.com/30600277/187520346-648d635f-c053-4e32-b519-7989bc08c89b.jpg)

Ich habe das Thema recherchiert und herausgefunden, daß diese Sachen im public/manifest.json definiert werden können (Logo, Standalone usw), aber das manifest.json ist in beiden Anwendungen gleich....

**Comment by mpagels** [comment]
Vllt. musst du, um `PWA` zu aktivieren, noch mehr machen? Dein repo ist nicht verlinkt, daher konnte ich nicht reinschauen. Aber schaue doch mal in diese Artikel rein:
- [https://nextjs.org/docs/faq](https://nextjs.org/docs/faq) -> dann `Can I make a Next.js Progressive Web App (PWA)?` auswählen
- https://medium.com/geekculture/how-to-make-a-next-js-app-a-pwa-a5e2b13da548
